doc/tutorial.rst

```diff
@@ -47,37 +47,37 @@ But let us begin with a slight detour...
 Sneak peek: what is going on under the hood
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. A top-level script (usually ``simple.py``, as in this tutorial) reads
+#. A top-level script (usually ``simple.py`` as in this tutorial) reads
    in an input file.
 
 #. Following the contents of the input file, a :class:`Problem
-   <sfepy.discrete.problem.Problem>` instance is created - this
+   <sfepy.discrete.problem.Problem>` instance is created -- this
    is the input file coming to life. Let us call the instance
-   ``problem``.
+   :class:`Problem <sfepy.discrete.problem.Problem>`.
 
```

Comment:
I would revert this, the instance should not be mixed with the class. Also below...